### PR TITLE
Add a DNS warning during install

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?

This PR will add a warning to the install dialogue if the domain is not 'localhost' reminding the user to set their DNS Records to point to their appwrite server.

Screenshot_2021-12-01_at_2 42 34_AM

Image from Gyazo

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes, I have.

### CODE REVIEW

**Comment by eldadfux** [comment]
@PineappleIOnic can you attach a gif/video of the setup process?

**Comment by PineappleIOnic** [comment]
@eldadfux I have added a GIF, sorry for the low quality. I'm not sure why it's so compressed.

**Comment by PineappleIOnic** [comment]
@christyjacob4

**Review by christyjacob4 — APPROVED** [review]
Looks good 👍

File: app/tasks/install.php [hunk 1]
```diff
@@ -167,6 +167,16 @@
             if(empty($input[$var['name']])) {
                 $input[$var['name']] = $var['default'];
             }
+
+            if ($var['filter'] === 'domainTarget') {
+                if ($input[$var['name']] !== 'localhost') {
+                    Console::warning("\nIf you haven't already done so, set the following record for {$input[$var['name']]} on your DNS provider:\n");
+                    $mask = "%-15.15s %-10.10s %-30.30s\n";
+                    printf($mask, "Type", "Name", "Value");
+                    printf($mask, "A or AAAA", "@", "<YOUR EXTERNAL IP>");
```
Comment (Meldiron) [inline app/tasks/install.php:176]:
```suggestion
                    printf($mask, "A or AAAA", "@", "<YOUR PUBLIC IP>");
```

Not sure if 'external' is the correct word here.

**Review by Meldiron — APPROVED** [review]
Looks good, one tiny suggestion.

File: app/tasks/install.php (hunk 1 above)
Comment (christyjacob4) [reply app/tasks/install.php:176]: Yeap Public is better 👍

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address Matej's comment and we can merge 👍

**Comment by PineappleIOnic** [comment]
> Please address Matej's comment and we can merge 👍

I've addressed the comment, We are ready to merge! 🥳

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**